content/md/user-management/troubleshoot-the-sso.md

```diff
@@ -0,0 +1,58 @@
+---
+id: troubleshoot-the-sso
+themes: user-management
+title: How to **troubleshoot** the Single Sign-On
+popular: false
+ee-only: true
+related: configure-saml-sso
+priority: low
+---
+
+# Overview
+
+Single Sign-On is a powerful tool for productivity but sometimes, as multiple applications are concerned, it can be a hassle to troubleshoot when something goes wrong with it.
+
+Here are a few tips to help you understand what is going on between your `IdP` and your `SP`.
```

Comment:
should we specify what IdP and SP stand for?

Reply:
Maybe put a link to the definitions you did in the other article. ;)

Reply:
Yep you're right. I will had a tip section about that.